packages/mgt-components/src/components/mgt-login/mgt-login.ts

```diff
@@ -537,9 +508,7 @@ export class MgtLogin extends MgtTemplatedComponent {
           .showPresence=${this.showPresence}
           .avatarSize=${displayConfig.avatarSize}
           line2-property="email"
-          class="person"
-        />
-       `
+          class="signed-in-person"/>`
```

Comment:
Well, this is my mistake to be fixed here. Web components are not self closing
```suggestion
          class="signed-in-person"
        ></mgt-person`
```